test/manifolds/symmetric_positive_definite.jl

```diff
@@ -6,23 +6,28 @@ include("../utils.jl")
     M2 = MetricManifold(SymmetricPositiveDefinite(3), Manifolds.LinearAffineMetric())
     M3 = MetricManifold(SymmetricPositiveDefinite(3), Manifolds.LogCholeskyMetric())
     M4 = MetricManifold(SymmetricPositiveDefinite(3), Manifolds.LogEuclideanMetric())
+    M5 = MetricManifold(SymmetricPositiveDefinite(3), Manifolds.BuresWassersteinMetric())
+    M6 = MetricManifold(
+        SymmetricPositiveDefinite(3),
+        Manifolds.GeneralizedBuresWassersteinMetric(2 * Matrix{Float64}(I, 3, 3)),
```

Comment:
Could we test it with a nondiagonal matrix here?

Reply:
Sure, I was just missing creativity here. Do you have a preferred non.diagonal matrix?

Reply:
I just tried an easy one that should be fine as well `[2.0 1.0 0.0; 1.0 2.0 1.0; 0.0 1.0 2.0]` which might be easier to debug code if necessary. I had to raise the `exp_log_atol_multiplier` a little bit in this case.

Reply:
OK, that's fine too :+1: